src/components/Apply.tsx

```diff
@@ -1,34 +1,43 @@
 import React from 'react';
 import { Helmet } from 'react-helmet';
+import { Container, Box, Typography } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
-import { Container, Box, Paper } from '@material-ui/core';
 
-/*
-Template for future page components.
-Use MUI's CSS-in-JS makeStyles() as shown below for more complex CSS/
-Otherwise, use <Box> components where possible for inline styling.
-Every page must have a Helmet tag for SEO purposes.
-*/
+import ApplyGrid from './ApplyGrid';
 
 const useStyles = makeStyles({
-  example: {
-    color: 'red',
+  heading: {
+    fontWeight: 'bold',
+    // background: '-webkit-linear-gradient(#3607F3 , #C3007A)',
```

Comment:
should these commented styles be removed?

Reply:
This is from when I was playing around with gradients, I'll just remove it for now

Reply:
Great, yeah we just don't need (code) commented in master i'd say